internal/install/install.go

```diff
@@ -139,6 +163,22 @@ func writeStackResources(elasticPackagePath string) error {
 	return nil
 }
 
+func writeDeployerResources(elasticPackagePath string) error {
```

Comment:
Nit: since this is specific to terraform (for now at least), consider renaming it to `writeTerraformDeployerResources`. We can generalize this later if/when we have another deployer that needs resources to be written.

Reply:
Fixed